libs/designer/src/lib/ui/mcp/panel/operation/EditOperationPanel.tsx

```diff
@@ -145,6 +141,13 @@ export const EditOperationPanel = () => {
       );
     }
 
+    LoggerService().log({
+      level: LogEntryLevel.Trace,
+      area: 'MCP.EditOperationPanel',
+      message: 'Edit operation details',
+      args: [`operationId:${selectedOperationId}`],
+    });
```

Comment:
area: MCP.EditOperation
message: Edit operation details saved
args: operationId and connectorId